ordering/__init__.py

```diff
@@ -67,6 +67,15 @@ def __contains__(self, item: _T) -> bool:
 
         return item in self._labels
 
+    def __eq__(self, other: object) -> bool:
+        if self is other:
+            return True
+        if not isinstance(other, Ordering):
+            return NotImplemented
```

Comment:
Do we want `order_a == order_b` if one is an `Ordering`, and the other a subclass, with the same items?